components/layout/client.coffee

```diff
@@ -34,6 +35,28 @@ module.exports = ->
   showLimitMessage()
   initConfirmableMessage()
   initLoggedOutCTA()
+  initGlobalBlockRouting()
+
+initGlobalBlockRouting = ->
+  # TODO: Extract and init block router only
+  # where we actually need it.
+  # Since migrating to Webpack, histories can conflict.
+  # Simply starting one history isn't working as it should,
+  # due to split packages (I believe).
```

Comment:
This is annoying. On the one hand, we shouldn't have this problem to begin with. On the other I can squint and imagine situations in which you might need two routers.